src-cpp/rdkafkacpp.h

```diff
@@ -81,13 +81,20 @@ typedef SSIZE_T ssize_t;
 
 /**@endcond*/
 
+#ifdef _MSC_VER    /* Visual Studio */
+#  pragma warning(disable : 4273)
```

Comment:
What's this for?

Reply:
To avoid this warning:
rdkafka_example.obj : error LNK2019: unresolved external symbol __imp_rd_kafka_mem_malloc referenced in function "public: __cdecl RdKafka::Headers::Header::Header(class RdKafka::Headers::Header const &)" (??0Header@Headers@RdKafka@@QEAA@AEBV012@@Z) [C:\projects\librdkafka\win32\rdkafka_example\rdkafka_example.vcxproj]
    13>rdkafka_example.obj : error LNK2019: unresolved external symbol __imp_rd_kafka_mem_free referenced in function "void __cdecl std::_Destroy_range<struct std::_Wrap_alloc<class std::allocator<class RdKafka::Headers::Header> > >(class RdKafka::Headers::Header *,class RdKafka::Headers::Header *,struct std::_Wrap_alloc<class std::allocator<class RdKafka::Headers::Header> > &)" (??$_Destroy_range@U?$_Wrap_alloc@V?$allocator@VHeader@Headers@RdKafka@@@std@@@std@@@std@@YAXPEAVHeader@Headers@RdKafka@@0AEAU?$_Wrap_alloc@V?$allocator@VHeader@Headers@RdKafka@@@std@@@0@@Z) [C:\projects\librdkafka\win32\rdkafka_example\rdkafka_example.vcxproj]
    13>